### DISCUSSION TITLE
Confirming version of Freesurfer easybuild to use in shared folder on Compute Canada

### DISCUSSION THREAD

**Discussion by dsodums** [McIntosh-Lab/tvb-ukbb#160] [Help] [answered]
Dear Experts, 
As I'm setting up a shared folder to have Freesurfer 7.10 accessible on Compute Canada for our group, I have a quick question. It looks like we need to upload an easybuild file from this link : https://github.com/ComputeCanada/easybuild-easyconfigs/tree/computecanada-master/easybuild/easyconfigs/f/FreeSurfer

There are two files for version 7.1 however:

FreeSurfer-7.1.0-centos6_x86_64.eb
FreeSurfer-7.1.0-centos7_x86_64.eb

Which one should we use?

Thanks,
Devin

**Comment by noahfl** [comment, marked as answer]
We use the FreeSurfer-7.1.0-centos6_x86_64.eb version. The centos7 one inexplicably wouldn't install for us so we reverted to the older one.

**Comment by dsodums** [comment]
Perfect, thanks for confirming!
